rendering_styles/default.render.xml

```diff
@@ -2545,6 +2545,10 @@
 				<case tag="attraction" value="train"/>
 				<case tag="attraction" value="log_flume"/>
 				<case tag="building" value="no" order="-1"/>
+				<switch moreDetailed="true" order="11" ignorePolygonArea="true" ignorePolygonAsPointArea="true">
```

Comment:
Normally polygon order should be in range 1-10. order="11" means that polygon will cover some ways.

Reply:
OK, thanks so much for this clarification.

I'll correct this to 10 in a few hours and will do again some testing, if this new value is also fine to you.

Why the polygon is not rendered at z14?

Reply:
Don't know. I need exact place location to test it.

Reply:
Rino Olmo Refuge: https://www.openstreetmap.org/way/120656176
Cassinelli Refuge: http://www.openstreetmap.org/way/120595368

Reply:
These objects were filtered by area during obf creation process and there is no easy way to fix it.

Reply:
Ah, I did not realize about the existence of such filter, I'll have a look at the code.

I thought that

    java -Djava.util.logging.config.file=loging.properties -Xms64M -Xmx512M -cp "./OsmAndMapCreator.jar;./lib/OsmAnd-core.jar;./lib/*.jar" net.osmand.binary.BinaryInspector -vmap -vmapobjects -bbox=10.0219093238,45.9338562315,10.0323629379,45.285758321 -zoom=10 Italy_lombardia_europe_2.obf

was just failing for some other reason (z10-), as strangely no feature at all is returned at that specific zoom (working fine instead at z11+).

This is the output at z10:
```
Binary index Italy_lombardia_europe_2.obf version = 2 edition = Wed Nov 01 01:00:00 CET 2017
1 Map data Italy_lombardia_europe - 195590432 bytes
        1.1 Map level minZoom = 15, maxZoom = 22, size = 119832438 bytes
                Bounds (left top - right bottom) : 5.0456, 48.4093 NE - 16.5914, 43.4945 NE
        1.2 Map level minZoom = 13, maxZoom = 14, size = 45800063 bytes
                Bounds (left top - right bottom) : 5.0456, 48.4093 NE - 16.5914, 43.4945 NE
        1.3 Map level minZoom = 12, maxZoom = 12, size = 21744576 bytes
                Bounds (left top - right bottom) : 5.0456, 48.4093 NE - 16.5914, 43.4945 NE
        1.4 Map level minZoom = 11, maxZoom = 11, size = 8178247 bytes
                Bounds (left top - right bottom) : 5.0456, 48.4092 NE - 16.5914, 43.4945 NE
        Total map objects: 0
2 Routing data Italy_lombardia_europe - 55347817 bytes
        Bounds (left top - right bottom) : 8.2653, 48.4093 NE - 16.5638, 44.5074 NE
3 Address data Italy_lombardia_europe - 25309012 bytes
        3.1 Address part size=8434728 bytes
        3.3 Address part size=9469239 bytes
        3.2 Address part size=2618490 bytes
4 Poi data Italy_lombardia_europe - 32302645 bytes
5 Transport data Italy_lombardia_europe - 1971393 bytes
        Bounds (left top - right bottom) : 8.7088, 46.5545 NE - 11.1244, 44.9818 NE
```

Does this mean that all features related to this bounding box are all filtered during obf creation for that specific zoom?

Reply:
Detailed maps contains no data at zoom 10. Look at the default map zoom levels in osmand map creator settings: 11;12;13-14;15-

Reply:
@xmd5a2 thanks again, now it is clear.